src/NuGet.Core/NuGet.Commands/Utility/AssetTargetFallbackUtility.cs

```diff
@@ -67,7 +67,7 @@ public static bool HasInvalidFallbackCombination(PackageSpec spec)
                 throw new ArgumentNullException(nameof(spec));
             }
 
-            return spec.TargetFrameworks.Any(e => e.Imports.Count > 0 && e.AssetTargetFallback.Count > 0);
+            return spec.TargetFrameworks.Any(e => e.PackageTargetFallbacks.Count > 0 && e.AssetTargetFallbacks.Count > 0);
```

Comment:
Rather minimize the change...this code will die in the future anyways as it will only go in P3. The original change Justin made will be kept in dev, has P4 and further.